pysr/sr.py

```diff
@@ -2586,3 +2583,122 @@ def _mutate_parameter(param_name: str, param_value):
         return False
 
     return param_value
+
+
+class PySRSequenceRegressor(PySRRegressor):
```

Comment:
Also, could this class be moved into a separate file? `sr.py` is too large as-is.